wiki/C++-Style-Guide.md

```diff
@@ -13,7 +13,7 @@
 
 # Background
 
-This style guide is also available on the [website](https://endless-sky.github.io/styleguide/styleguide.xml).
+This style guide is also available on the [website](https://endless-sky.github.io/styleguide/styleguide.xml). However, in case of inconsistencies, the version here applies.
```

Comment:
Do we want to keep it there? https://github.com/endless-sky/endless-sky.github.io/pull/66

Reply:
If we have that PR ready, then probably not.